ynr/apps/moderation_queue/templates/moderation_queue/photo-review.html

```diff
@@ -90,6 +90,56 @@ <h3>Click and drag in the image to crop</h3>
       </p>
     </div>
 
+    <h3>Rotate the image as needed</h3>
+    <input type="hidden" name="queued_image_id" value="{{ queued_image.id }}">
+    <button name="rotate" type="submit" value="left">Anti-clockwise ⟲</button>
+    <button name="rotate" type="submit" value="right">Clockwise ⟳</button>
```

Comment:
If you want to save some space, you could add `class="tiny"` to these buttons and chnage the `h3` to a `p` and wrap the whole lot in a `<div class=col-6>[text]</div><div class=col-6>[buttons]</div>`.

(untested)